src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -194,253 +328,349 @@ const CurationPage = (props: ICurationPageProps) => {
+                                            location={`${CANCER_TYPE_THERAPY_INDENTIFIER}${getMutationName(mutation)}, ${cancerTypeName}, ${
+                                              treatment.name
+                                            }`}
+                                          />
+                                        }
+                                        name="evidenceDescription"
+                                      />
+                                      <div className="mb-2">
+                                        <AutoParseRefField summary={treatment.description} />
+                                      </div>
+                                    </Collapsible>
+                                  );
+                                })
+                              );
+                            }, [])}
+                          </Collapsible>
+                        );
+                      })}
+                  </Collapsible>
+                </Col>
+              </Row>
+            ))}
           </div>
         )}
         <VusTable hugoSymbol={hugoSymbol} />
+        <Modal isOpen={showFilterModal} toggle={handleToggleFilterModal}>
+          <ModalHeader>
+            <Container>
+              <Row>
+                <Col>Filters</Col>
+              </Row>
+            </Container>
+          </ModalHeader>
+          <ModalBody>
+            <Container>
+              <h6 className="mb-2">Oncogenicity</h6>
+              <Row>
+                {oncogenicityFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setOncogenicityFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
+                    </InputGroup>
+                  </Col>
+                ))}
+              </Row>
+              <h6 className="mb-2 mt-2">Mutation effect</h6>
+              <Row>
+                {mutationEffectFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setMutationEffectFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
+                    </InputGroup>
+                  </Col>
+                ))}
+              </Row>
+              <h6 className="mb-2 mt-2">Therapeutic implication level of evidence</h6>
```

Comment:
Maybe could shorten to just Therapeutic Levels?

Reply:
Agree with Calvin